Users/Install.rst

```diff
@@ -159,7 +159,14 @@ can run
 
 ::
 
-    (venv)$ pip3 install coala-bears --pre
+    (venv)$ git clone https://github.com/coala/coala
+    (venv)$ cd coala/coala
+    (venv)$ pip3 install .
+    (venv)$ cd -
+    (venv)$ git clone https://github.com/coala/coala-bears
+    (venv)$ cd coala/coala-bears
+    (venv)$ pip3 install .
```

Comment:
yeah, I expect people to know about pip install and git clone, cd is rather trivial as well.